src/functions/apis/auth/action.ts

```diff
@@ -13,11 +22,30 @@ export class AuthAction {
         this.repository = connection.getCustomRepository(UserRepository);
     }
 
-    async execute(username: string, password: string): Promise<UserModel> {
+    async execute(username: string, password: string): Promise<LoginResult> {
         const user = await this.repository.getUserByUserName(username);
         if (!user) throw new AuthAccessDenied();
 
         if (!bcrypt.compareSync(password, user.password)) throw new AuthAccessDenied();
-        return user;
+
```

Comment:
Please make the generation of the access token and refresh token part of the TokenService

Reply:
resolved, please check new commit thanks